src/js/hmm/components/HMMDetail.tsx

```diff
@@ -120,7 +116,9 @@ export default function HMMDetail({ match }: HMMDetailProps) {
 
                         <tr>
                             <th>Best Definitions</th>
-                            <td>{names}</td>
+                            <td>
+                                <div className="flex gap-1 leading-none">{names}</div>
+                            </td>
```

Comment:
Does this not work?
```suggestion
                            <td className="flex gap-1 leading-none">
                                {names}
                            </td>
```

Reply:
That doesn't work but I just applied the styling to the label instead